cg/services/order_validation_service/models/errors.py

```diff
@@ -57,7 +57,12 @@ class OccupiedWellError(CaseSampleError):
 
 class ApplicationNotValidError(CaseSampleError):
     field: str = "application"
-    message: str = "Application is not valid"
+    message: str = "Chosen application does not exist"
+
+
+class ApplicationNotCompatibleError(CaseSampleError):
+    field: str = "application"
+    message: str = "Application is not compatible with the chosen workflow"
```

Comment:
A clearer message might be `Application is not allowed for the chosen workflow`.